app/src/main/java/com/example/sharingang/items/Item.kt

```diff
@@ -1,6 +1,23 @@
 package com.example.sharingang.items
 
+import java.util.*
+
 /**
  * Item represents an item available for sharing, or for sale.
  */
-data class Item(val description: String)
+data class Item(
+    val title: String = "",
+
+    val description: String = "",
+
+    /** URLs of the images */
+    val images: List<String> = listOf(),
+
+    /** Price in cents */
+    val price: Int = 0,
+
+    val createdAt: Date = Date(),
+
+    /** ID generated by the database */
+    var id: String? = null
```

Comment:
I'm unsure that we want to allow `title` and `description` to default to `""`. I can't imagine a situation where an item without a title would be valid. Description, sure, but we probably should require it to be passed anyway.

For price `0` as a default does make sense, because we do have sharing of many items without a sale, so yeah.

Reply:
You're right it doesn't make sense to have an object without a title.
I had to do this because Firebase requires having a no-argument constructor to deserialize it with `.toObject(Item::class.java)`.
Can we keep it like this for now and maybe add manual deserialization when we have more complicated fields (e.g. geo coordinates)?

Reply:
Ah, yeah that makes sense. Sure, it's fine for now, just a minor point.